Source/Core/IndexDatatype.js

```diff
@@ -95,7 +95,7 @@ define([
      * or <code>Uint32Array</code> depending on the number of vertices.
      *
      * @param {Number} numberOfVertices Number of vertices that the indices will reference.
-     * @param {*} indicesLengthOrArray Passed through to the typed array constructor.
+     * @param {Object} indicesLengthOrArray Passed through to the typed array constructor.
```

Comment:
This should be a `number` or `Array`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint32Array